configure-ironic.sh

```diff
@@ -149,7 +106,9 @@ IRONIC_CONFIG_OPTIONS="--config-file /etc/ironic/ironic.conf"
 configure_client_basic_auth() {
     local auth_config_file="/auth/$1/auth-config"
     if [ -f ${auth_config_file} ]; then
-        IRONIC_CONFIG_OPTIONS+=" --config-file ${auth_config_file}"
```

Comment:
The option should be removed completely and ironic-dist.conf should also be considered.

Reply:
I have removed the option. About ironic-dist.conf, since it is used by ironic by default, so I guess we don't need to do anything.